### PR TITLE
Thermal Express driver

### PR DESCRIPTION
**Pull request by gmdimitriz**
Please check it again....

### CODE REVIEW

**Comment by MaxKellermann** [comment]
./src/NMEA/Info.hpp:46:1: error: 'NMEAInfo' defined as a struct here but previously declared as a class [-Werror,-Wmismatched-tags]

File: src/Device/Driver/ThermalExpress/Driver.hpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+}
+*/
+#ifndef XCSOAR_THERMALEXPRESS_DRIVER_HPP
+#define XCSOAR_THERMALEXPRESS_DRIVER_HPP
+
+#include "Device/Driver/ThermalExpress.hpp"
+#include "Device/Driver.hpp"
+#include "Device/Port/Port.hpp"
+#include "NMEA/Info.hpp"
```
Comment (MaxKellermann) [inline src/Device/Driver/ThermalExpress/Driver.hpp]: Did you fix that compiler warning by changing the forward declaration back to an `#include`? Sigh.

File: src/Device/Driver/ThermalExpress/Driver.hpp [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+class NMEAInputLine;
+
+class ThermalExpressDevice : public AbstractDevice {
+
+private:
+  bool ParseTXP(NMEAInputLine &line, NMEAInfo &info);
+
+public:
+  bool ParseNMEA(const char *line, struct NMEAInfo &info) override;
```
Comment (MaxKellermann) [inline src/Device/Driver/ThermalExpress/Driver.hpp]: Superfluous `struct` keyword.

Comment (gmdimitriz) [inline src/Device/Driver/ThermalExpress/Driver.hpp]: removed

**Pull request closed without merging**